### DISCUSSION TITLE
Attributter - Lokasjon "location"

### DISCUSSION THREAD

**Discussion by eirikbroen** [NorskHelsenett/Tillitsrammeverk#10] [General]
Bør ta en diskusjon på hva som er forventning knyttet til verdi på dette attributtet. I hvilke tilfeller gir det merverdi utover "child-org" å spesifisere lokalitet? Kan se for meg flere situasjoner hvor det for eksempel ikke er fornuftig å spesifisere avdeling - Eks. Overgrepsmottak. Hva er formålet med attributtet? Er det hovedsaklig å gjøre det enklere for pasient å forstå innsynslogg? Kanskje et attributt som gir større verdi i spesialistsektor hvor avdeling på større sykehus er viktig. For Oslo kommune vil nok verdi på dette attributtet være lik virksomhet representert ved _point_of_care_name_.

**Comment by steinarnoem** [comment]
Vi skiller oss litt fra EHDSI i denne spesifikasjonen ved at vi bruker "point_of_treatment" for å angi behandlingssted, og "locality" for å angi det fysiske stedet - som kan være en adresse, en avdeling osv..  For øvrig er attributtet definert som fritekst, som betyr at vi må ha strenge regler rundt gyldig tegnsett..

En mulig tilnærming er at vi gjør "locality" til et valgfritt informasjonselement, slik at det er opp til EPJ/konsument å legge ved dette attributtet. I dagens utkast er også "point_of_care" valgfritt, så kanskje gir det mening at begge disse informasjonselementene ikke er påkrevd?

**Reply by OK-SverreM** [reply]
For oss i Oslo kommune (og jeg vil tippe andre kommuner) er det "point_of_care" / behandlingssted som er det aktuelle og relevante å oppgi - for eksempel legevakt, KAD, sykehjem etc. 
Så har jeg forstått fra dialogene vi har hatt med HSØ at de ser nytte i å legge til noe mer spesifikt, f.eks. hvilken avdeling på den aktuelle klinikken hvor behandlingen ble utført. Men jeg tror vel de også alltid vil kunne angi en organisasjonsenhet som "point_of_care"? 
Dette bør dobbeltsjekkes med spesialistsiden, men jeg har tro på at "point_of_care" kan være obligatorisk - mens "locality" ikke kan / bør være det.

**Reply by OK-SverreM** [reply]
Ut fra gårsdagens møte i **Feature-team for informasjons- og datamodell** forsto jeg det slik at hypotesen min over stemmer - både spesialisthelsetjenesten og kommunene vil alltid kunne oppgi behandlingssted (med organisasjonsnummer).
**_Men_**: Det vil ikke nødvendigvis gjelde for fastlegene. Vi kan da velge mellom følgende tilnærminger:

1. Gjøre attributtet obligatorisk, og fylle ut samme organisasjonsnummer to ganger, i to ulike attributter, for fastlegene.
2. Gjøre attributtet ikke-obligatorisk, men pålegge (i rutiner / vedlegg til bruksvilkår) sykehus og kommuner å fylle ut dette attributtet.

Jeg heller i retning av alternativ 2. 
Det er ikke god semantisk skikk å oppgi samme id for to attributter som egentlig skal beskrive to ulike ting, + at det er liten vits i å duplisere informasjon i sikkerhetsbilletten?

**Comment by steinarnoem** [comment]
Temaet som dreier seg om hvor pasienten befinner seg diskuteres i en annen tråd - derfor lukker vi denne tråden..
